src/DIRAC/RequestManagementSystem/Client/ReqClient.py

```diff
@@ -309,8 +313,7 @@ def finalizeRequest(self, requestID, jobID, useCertificates=True):
         :param str requestID: request id
         :param int jobID: job id
         """
-
-        stateServer = JobStateUpdateClient(useCertificates=useCertificates)
+        stateServer = _JobDBInteraction(useCertificates)
```

Comment:
you're going to run into the same issue with the JobMonitoringClient below.

Reply:
I know but that is used in multiple places so I thought it was better to focus on one client at a time.